app/assets/stylesheets/partials/_modal.scss

```diff
@@ -0,0 +1,84 @@
+#modalOverlay {
+  background-color: white;
+  display: none; 
+  height: 100%;
+  left: 0;
+  margin: 0;
+  opacity: 0.7;
+  padding: 0;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 9999;
+}
+
+#modal {
+  background-color: white;
+  border: 2px solid #C9D4D9;
+  border-radius: 6px;
+  display: none;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 20px;
+  position: absolute;
+  top: 40px;
+  left: 25%;
+  width: 50%;
+  z-index: 99999;
+
+  @media (max-width: 767px) {
+    left: 15%;
+    width: 70%;
+  }
+
+  @media (max-width: 767px) {
+    left: 10%;
+    width: 80%;
+  }
+
+  h3 {
+    font-size: 15px;
+    font-weight: normal;
+    padding-bottom: 15px;
+  }
+
+  hr {
+    border: 1px solid #EEF2F4;
+  }
+
+  #modalCloseButton {
+    color: #0069A0;
+    cursor: pointer;
+    font-size: 11px;
+    line-height: 1.1;
+    position: absolute;
+    right: 20px;
+    text-align: right;
+    top: 20px;
+    width: 80px;
+    
+    span {
+      color: #9A9A9A;
+      display: block;
+
+      &#cancel {
+
+        background-color: #0069A0;
+        border-radius: 3px;
+        color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.